packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -29,6 +29,8 @@ export interface SearchInputProps extends Omit<React.HTMLProps<HTMLDivElement>,
   value?: string;
   /** Array of attribute values */
   attributes?: string[] | SearchAttribute[];
+  /* Additional elements added after the attributes in the form */
```

Comment:
Could you add the suggestion to wrap additional form elements in a FormGroup component for formatting?

Reply:
Fixed, let me know if you have a better wording suggestion :)

Reply:
Looks good. Could you also update the advanced form unit test to include testing this new prop?

Reply:
definitely, forgot about the tests, sorry !